driver/src/contracts/stablex_contract.rs

```diff
@@ -189,6 +200,29 @@ impl<'a> StableXContract for StableXContractImpl<'a> {
 
         Ok(())
     }
+
+    fn stream_events(
+        &self,
+    ) -> BoxStream<'static, Result<Event<batch_exchange::Event>, ExecutionError>> {
+        Box::pin(
+            self.instance
+                .all_events()
+                .from_block(BlockNumber::Earliest)
```

Comment:
Should this be `latest` since it doesn't retrieve past events anyways? I find it would better document what the stream is actually producing.